_pages/en/it-picture-long-term.md

```diff
@@ -38,59 +45,47 @@ They can also use the digital services on behalf of clients.
-- TODO: Add source (Microsoft)
+## Private Sector
 
-User privacy must still be preserved and modern digital identity recognition technologies are leveraged (e.g. facial, biometrics).
+### Any platform, any device, 3rd party integration
 
-For more details on its implementation, please refer to the [Business section](##Businesses)
+Any platform or device can be integrated with GC and ESDC services and development is often done in partnership with the private sector.
+This is possible because ESDC services are available as APIs that can be used by third party platforms and services (Banks, CRA, Social media, Job boards, Internet of things, ..).
+Employers, Banks, and Credit Unions are the new front lines of ESDC as they are now provided with technological means to interact with ESDC services.
 
-> “Automatic enrollment reduces the burden on many seniors by enabling them to receive their Old Age Security pension and Guaranteed Income Supplement at age 65 without ever having to apply for them. We continue our efforts to modernize the Old Age Security and Guaranteed Income Supplement services to meet the needs of Canadians.”
-> > ––The Honourable Jean-Yves Duclos, Minister of Families, Children and Social Development
+Ecosystem of services are created by leveraging private sector expertise where opportunities exist (win-win public-private partnerships), e.g.:
+- Banks detecting missed paycheck, job websites targeting people who are job hunting...
+- Enhanced security through integration in private sector service (e.g., banks detecting fraudulent transactions)
 
-## Private Sector
+### Open Collaboration
 
-Any platform or device can be integrated with GC and ESDC services.
-This is possible because ESDC services are available as APIs that can be used by third party platforms and services (Banks, CRA, Social media, Job boards, Internet of things, ..).
-
-- Create ecosystem of services by leveraging private sector expertise where opportunities exist (win-win public-private partnerships)
-  - E.g., banks detecting missed paycheck, job websites targeting people who are job hunting
-  - Enhanced security through integration in private sector service (e.g., banks detecting fraudulent transactions)
+- All of ESDC's custom built software are published as open source, allowing Academia and learning institutions to use them, helping grow technological skill sets in the country, fostering economic growth, improving transparency and trust in government, and increasing the breadth of talent for ESDC to use by welcoming contributions back
+- ESDC data (performance and automated decision making statistics) published as open datasets
 
 ## IITB-ESDC
 
 ### Modern Open Organization
 
 IITB is a modern open organization with a very open culture.
 Wherever possible code and projects are developed in the open and use of open source software is the norm.
+IITB is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
```

Comment:
What does this look like? 

- IITB is responsible for providing the automated services required to run a secure and agile business. This includes, automated software onboarding approval, key data exchange systems with which business may interact with, and an environment by which business may securely deploy their own solutions

Reply:
added!